composite/outlier-detection/src/config.ts

```diff
@@ -0,0 +1,19 @@
+import { logger } from '@chainlink/external-adapter'
+import { util } from '@chainlink/ea-bootstrap'
+import { Config } from '@chainlink/types'
+
```

Comment:
Here you can make a specific Config type for this adapter, one that extends our base Config type.

```suggestion
import types from '@chainlink/types'

export type Config = types.Config & {
   threshold: {
     checks: number
     onchain: number
   }
}

```